docs/content/guides/dagster/recommended_project_structure.mdx

```diff
@@ -0,0 +1,202 @@
+
+- Note: Certain sensors like [run status sensors](/concepts/partitions-schedules-sensors/sensors#run-status-sensors) can listen to multiple jobs and do not trigger a job. We recommend keeping these sensors in the repository definition as they are often for alerting and monitoring at the repository level.
+
+### For resources
+
+Make resources reusable and share them across jobs or asset groups.
+
+In this example, resources (e.g., database connections, Spark sessions, API clients, and I/O managers) are grouped in the [`resources/__init__.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/__init__.py) file, where they are bound to configuration sets which vary based on environment. In complex projects, we find it helpful to make resources reusable and configured with pre-defined values via <PyObject object="configured" />. So when developing a project, instead of trying to find their own way to model external components, teammates can all quickly get stuff done using a pre-defined resource set, or make changes to resources that are shared among the team.
+
+This pattern also helps you to easily execute jobs in different environments without code changes. In this example, we dynamically define a repository based on the deployment in [`repository.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/repository.py) and are able to keep all code the same across testing, local development, staging, and production. Read more about our recommendations in [Transitioning Data Pipelines from Development to Production](/guides/dagster/transitioning-data-pipelines-from-development-to-production)
+
+### For ops and graphs
+
+
+Although assets are useful in many ways, [ops](/concepts/ops-jobs-graphs/ops) and [graphs](/concepts/ops-jobs-graphs/graphs), are part of the Dagster ecosystem and can be integrated into your project structure.
+
+If you have use cases that don't meet the criteria for using assets, you may still use ops and graphs to build out these use cases. For example:
+
+- Sending an email or Slack message on a regular basis to users 
+- Reading a file of user IDs and change the value of a particular attribute for each user 
+- Scanning a data warehouse for tables that haven't been used in months and delete them
+
+In these cases, we recommend keeping all ops together in an 'ops_graphs/' directory. Similarly to assets, a structure under 'ops_graphs/' can be used to group jobs and their respective graphs into distinct business domains.  
```

Comment:
tbh this was where i got stuck last time cos i wasn't v happy with the recommendations i came up with.

> Similarly to assets, a structure under 'ops_graphs/' can be used to group jobs and their respective graphs into distinct business domains.
for collaboration and code reading/finding purpose, ops/graphs are similar to assets, but they are also different because assets come with "load_assets_from..." which makes asset loading depends on the folder structure but op/graph won't. 


@sryza - would love to get your thoughts on how one might structure the ops and graphs.